src/components/ChecWysiwyg.vue

```diff
@@ -0,0 +1,571 @@
+          :class="{ 'menubar__button--is-active': isActive.bullet_list() }"
+          @click="commands.bullet_list"
+        >
+          <ChecIcon icon="bullet-list" size="base" />
+        </button>
+        <button
+          class="wysiwyg__menubar--button"
+          :class="{ 'wysiwyg__menubar--button--is-active': isActive.blockquote() }"
+          @click="commands.blockquote"
+        >
+          <ChecIcon icon="quote" size="base" />
+        </button>
+      </div>
+    </EditorMenuBar>
+  </div>
+</template>
+
+<script>
+import { Editor, EditorContent, EditorMenuBar } from 'tiptap';
+import {
+  Blockquote,
+  Heading,
+  BulletList,
+  Bold,
+  Italic,
+  ListItem,
+} from 'tiptap-extensions';
+import ChecIcon from '@/components/ChecIcon.vue';
+import uniqueId from '@/lib/helpers/createUniqueId';
+
+export default {
+  name: 'TextField',
+  components: {
+    ChecIcon,
+    EditorContent,
+    EditorMenuBar,
+  },
+  inheritAttrs: false,
+  props: {
+    /**
+     * The state of the text field. One of "disabled", "error".
+     */
+    variant: {
+      type: String,
+      default: '',
+    },
+    /**
+     * The value the v-model directive utilizes
+     * @see https://vuejs.org/v2/guide/components.html#Using-v-model-on-Components
+     */
+    value: {
+      type: [String, Number],
+      default: '',
+    },
+    /**
+     * Class to pass to inner input element
+     */
+    innerInputClass: {
+      type: String,
+      default: '',
+    },
+    /**
+     * Label for input
+     */
+    label: {
+      type: String,
+      default: '',
+    },
+    /**
+     * An optional placeholder which will appear below the label as the current value when the field has no value.
+     * Appears like a value in the field, but acts like a traditional placeholder.
+     */
+    placeholder: String,
+    /**
+     * Additional input attributes that should be applied to the native input
+     */
+    additionalInputAttributes: {
+      type: Object,
+      default: () => ({}),
+    },
+    /**
+     * Allows for toggling of html attribute of "required" in label
+     */
+    required: Boolean,
+    /**
+     * The style variant for the text Input. One of "light" or "dark"
+     */
+    styleVariant: {
+      type: String,
+      default: 'light',
+      validator(value) {
+        return ['light', 'dark'].includes(value);
+      },
+    },
+  },
+  data() {
+    const { input, ...nonInputListeners } = this.$listeners;
+    return {
+      isScrollable: false,
+      isScrolledToTop: true,
+      slotObserver: null,
+      id: uniqueId(this.name, this.value, 'chec-switch')(),
+      slotWidth: 0,
+      hasSlot: false,
+      editor: new Editor({
+        extensions: [
+          new Blockquote(),
+          new BulletList(),
+          new Heading({ levels: [1, 2, 3] }),
+          new Bold(),
+          new Italic(),
+          new ListItem(),
+        ],
+        content: '<p>This is just a boring paragraph</p>',
```

Comment:
```suggestion
        content: '',
```

Seems like something that should be in the storybook